frontend/src/metabase/css/components/buttons.module.css

```diff
@@ -9,19 +9,19 @@
   padding: 0.5rem 0.75rem;
   background: transparent;
   border: 1px solid color-mix(in srgb, var(--mb-color-border), black 5%);
-  color: var(--mb-color-text-dark);
+  color: var(--mb-color-text-primary);
   cursor: pointer;
   font-weight: bold;
   font-family: var(--mb-default-font-family), sans-serif;
   border-radius: var(--default-button-border-radius);
+  transition: all 200ms linear;
+  transition-property: color, border-color, background-color;
```

Comment:
This doesn't actually change the button behavior, but rather allow, the overridden button to override this transition (`frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePickerShortcuts.styled.tsx`).

Because if you hover over "Date all options" filter, you'll notice slight delay when the hover color change, and this doesn't happen to other dropdowns, so this change is for UX consistency.